cdap-ui/app/directives/epoch/charts.js

```diff
@@ -33,6 +33,7 @@ ngEpoch.controller('epochController', function ($scope, $compile, caskWindowMana
     }, options);
 
     angular.extend(options, forcedOpts || {}, {
+      // TODO: Do we need to copy $scope.history here? I think not.
```

Comment:
I didn't think $scope.history would have any data upon init.
Can we remove this? (Epoch chart directive).